cms/static/js/views/content_group_details.js

```diff
@@ -0,0 +1,28 @@
+define([
+    'js/views/baseview'
+], function(BaseView) {
+    'use strict';
+
+    var ContentGroupDetails = BaseView.extend({
```

Comment:
I'd call this ```ContentGroupDetailsView``` as it isn't the details for the content group, it is the view showing the details.